matchms/Spectrum.py

```diff
@@ -72,30 +83,31 @@ def __init__(self,
             Array of intensities for the peaks
         metadata
             Dictionary with for example the scan number of precursor m/z.
+        metadata_harmonization : bool, optional
+            Set to False if default metadata filters should not be applied.
+            The default is True.
         """
-        if metadata is None:
-            self._metadata = {}
-        else:
-            self._metadata = metadata
-        self.peaks = Spikes(mz=mz, intensities=intensities)
+        self._metadata = Metadata(metadata)
+        if metadata_harmonization is True:
+            self._apply_metadata_harmonization()
+        self.peaks = Fragments(mz=mz, intensities=intensities)
```

Comment:
I assume the `retention_time` and `retention_index` harmonization should go in here? Or what exactly did you mean?